### ISSUE TITLE
sudo docker-compose logs -f   wouldn't work

### ISSUE THREAD

**Issue by gmorningman** [ergolabs/ergo-dex-backend#26]
I didn't copy paste exactly what the error was but it was something along the lines of [ERROR DEX_PATH /modules/blah/blah/blah]

I edit my docker-compose.yml to just use my full path and its working fine now.  however didn't work "out of the box"
/home/username/ergo/ergo-dex-backend/modules/utxo-tracker/target/docker/stage
/home/username/ergo/ergo-dex-backend/modules/amm-executor/target/docker/stage
/home/username/ergo/ergo-dex-backend/modules/pool-resolver/target/docker/stage

Brand new installation of ubuntu server 20.04.3

**Comment by FidgetyRat** [comment]
Yes, the environment variable set in the script isn't working even if explicitly exported first (Same for me on Debian). Had to hard code the paths.

**Comment by riidatibula** [comment]
Try using the latest version of [docker-compose](https://github.com/docker/compose/releases). I had the same error when using the earlier versions (1.29.x)
